packages/terra-dialog-modal/tests/wdio/dialog-modal-spec.js

```diff
@@ -74,4 +74,26 @@ describe('Dialog Modal', () => {
 
     Terra.it.matchesScreenshot('focused shifted outside the end of the modal', { selector: '#root' });
   });
+
+  describe('Closing Select inside modal using escape', () => {
+    it('Opens the Select inside the modal', () => {
+      browser.url('/#/raw/tests/terra-dialog-modal/dialog-modal/diaglog-modal-with-select');
+      browser.click('#trigger-dialog-modal');
+      browser.click('[data-terra-form-select-toggle]');
+    });
+
+    Terra.it.matchesScreenshot('Select Opened inside a modal', { selector: '#root' });
+
+    it('Closes only the select on pressing Escape', () => {
+      browser.keys(['Escape']);
+    });
+
+    Terra.it.matchesScreenshot('Select Closed', { selector: '#root' });
+
+    it('Closes the modal on pressing Escape', () => {
+      browser.keys(['Escape']);
+    });
+
+    Terra.it.matchesScreenshot('Modal Closed', { selector: '#root' });
```

Comment:
`Terra.it.matchesScreenshot` is being deprecated. It will be replaced by `Terra.validates.screenshot`.

```suggestion
    it('Opens the Select inside the modal', () => {
      browser.url('/#/raw/tests/terra-dialog-modal/dialog-modal/diaglog-modal-with-select');
      browser.click('#trigger-dialog-modal');
      browser.click('[data-terra-form-select-toggle]');
      Terra.validates.screenshot('Select Opened inside a modal', { selector: '#root' });
    });

    it('Closes only the select on pressing Escape', () => {
      browser.keys(['Escape']);
      Terra.validates.screenshot('Select Closed', { selector: '#root' });
    });

    it('Closes the modal on pressing Escape', () => {
      browser.keys(['Escape']);
      Terra.validates.screenshot('Modal Closed', { selector: '#root' });
    });
```